gradle/scripts/detekt.yml

```diff
@@ -35,6 +34,10 @@ potential-bugs:
     UnsafeCast:
         active: false
 
+performance:
+    ArrayPrimitive:
+        active: false
```

Comment:
There were three new kinds of issues in your code (mainly in test cases), which you have to decide what to do about them ^^